src/app/modules/map/ol/lib/resources/layer-livelayer.component.ts

```diff
@@ -111,22 +115,158 @@ export class FreeboardLiveLayerComponent
     this.clearlayers();
   }
 
+  /** Refresh time dimension bounds from WMS/WMTS capabilities */
+  private async refreshTimeDimension(layerId: string, liveLayer: LiveLayer) {
```

Comment:
Time dimension processing should be happening outside of the live-layer component.
The component's purpose is to render the layer.... dimension changes should be passed to the component.